tools/generators/dh-domain-generator/schema.json

```
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "dh-domain-generator",
+  "description": "Generate an entire domain that's going to be used in DataHub.",
+  "type": "object",
+  "properties": {
+    "domain": {
+      "type": "string",
+      "description": "What is the name of the domain?",
+      "$default": {
+        "$source": "argv",
+        "index": 0
+      },
+      "x-prompt": "What is the name of the domain?"
+    },
+    "name": {
+      "type": "string",
+      "description": "What is the name of the feature?",
+      "x-prompt": "What is the name of the feature?"
```

suggestion: Maybe too explicit but to explain a bit why a feature name is needed. Open for other suggestions too or keeping the current value, if you think it's fine enough.

```suggestion
      "description": "What is the name of the first feature under the new domain?",
      "x-prompt": "What is the name of the first feature under the new domain?"
```

---

I like it!